atomix/cluster/src/test/java/io/atomix/raft/RaftRule.java

```diff
@@ -623,4 +597,31 @@ public long awaitCommit() throws Exception {
       return commitFuture.get(30, TimeUnit.SECONDS);
     }
   }
+
+  private final class RaftSnapshotListener implements PersistedSnapshotListener {
```

Comment:
This was just again reordering? Seems that there is still something wrong with my setup :sweat_smile: